### ISSUE TITLE
UI Component listing action column outside of screen when too many columns

### ISSUE THREAD

**Issue by milansimek** [magento/magento2#14153]
### Preconditions

1. Magento 2.x
2. Google Chrome 65 (maybe previous versions also)

### Steps to reproduce

1. Edit cms_page_listing.xml
2. Add the following snippet to the columns tag:
```xml
       <column name="text1">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text2">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text3">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text4">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text5">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text6">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text7">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text8">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text9">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text10">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text11">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text12">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
        <column name="text13">
            <settings>
                <label translate="true">Text</label>
                <visible>true</visible>
            </settings>
        </column>
```

3. Clear cache and empty `ui_bookmark` table
4. Go to CMS page listing
5. Resize screen to 1200px wide
6. Scroll to the right to the actions column
7. Click on it

### Expected result

The dropdown menu should be positioned under the link

### Actual result

Dropdown menu is out of screen

![image](https://user-images.githubusercontent.com/10584134/37549670-cc229fb8-2982-11e8-952b-144f2bf9438d.png)

**Comment by magento-engcom-team** [comment]
@milansimek, thank you for your report.
We've acknowledged the issue and added to our backlog.

**Comment by saurabh-parekh** [comment]
I am working on this at #dmcdindia

**Comment by magento-engcom-team** [comment]
@saurabh-aureate thank you for joining. Please accept team invitation [here](https://github.com/orgs/magento/invitation) and self-assign the issue.

**Comment by hitesh-wagento** [comment]
I am working on this

**Comment by magento-engcom-team** [comment]
Hi @milansimek. Thank you for your report. 
The issue has been fixed in magento/magento2#15459 by @hitesh-wagento in 2.2-develop branch
Related commit(s):
 - [2cc78bcd1895c47e958e2bb2799a8694d30220fa](https://github.com/magento/magento2/commit/2cc78bcd1895c47e958e2bb2799a8694d30220fa)

The fix will be available with the upcoming 2.2.6 release.

**Comment by sidolov** [comment]
Hi @milansimek. Thank you for your report. 
The issue has been fixed in magento/magento2#15746 by @vgelani in 2.3-develop branch
Related commit(s):
 - [365bfb4c81914041d063d99b3a87fe8d96fcc6c9](https://github.com/magento/magento2/commit/365bfb4c81914041d063d99b3a87fe8d96fcc6c9)

The fix will be available with the upcoming 2.3.0 release.
